extensions/yafw/src/optimize.tsx

```diff
@@ -0,0 +1,56 @@
+import { Toast as RaycastToast, showHUD } from "@raycast/api";
+import { Ffmpeg, FfmpegBinaryNotFoundException } from "./objects/ffmpeg";
+import { FfmpegVideo } from "./objects/ffmpeg.video";
+import { Ffprobe, FfprobeBinaryNotFoundException } from "./objects/ffprobe";
+import { FinderIsNotFrontmostAppException, SelectedFinderFiles } from "./objects/selected-finder.files";
+import { Toast } from "./objects/toast";
+
+export default async function Command(props: { arguments: { preset: "smallest-size" | "optimal" | "best-quality" } }) {
```

Comment:
isn't this exactly what the "compress video" command does?

Reply:
@pablopunk sort of, but without UI. Maybe it is possible to call such operations something else. I want to leave quick access for users, because I too often need to run a command without any additional action. You know, “fire and forget”.
Please let me know if you have any suggestions on this. I don't have good solution for now.

Reply:
that's fair, it was a nice experience to use it; let's leave it like that and I'll try to unify the whole thing later (without sacrificing these non-ui commands)